test/duckdb_test/column_test.rb

```diff
@@ -145,6 +145,11 @@ def test_logical_type
       assert(logical_types.all? { |logical_type| logical_type.is_a?(DuckDB::LogicalType) })
     end
 
+    def test_logical_type_type
```

Comment:
I'm thinking about where we should test this method because test data will be duplicated with `test/duckdb_test/column_test.rb`. Should we define it in `test/duckdb_test/logical_type_test.rb`?

Reply:
@otegami 

This test is for DuckDB::LogicalType#type, right?
Then, I think it should be in test/duckdb_test/logical_type_test.rb.

Reply:
Thank you so much. I think it should be there after rethinking it now too.